modules/govuk_jenkins/manifests/packages/govuk_python.pp

```diff
@@ -25,10 +25,13 @@
     key          => $apt_mirror_gpg_key_fingerprint,
   }
 
-  package { 'govuk-python':
-    ensure  => $govuk_python_version,
-    require => Apt::Source['govuk-python'],
-  }
+  ensure_packages(
+    ['govuk-python'],
```

Comment:
did you mean govuk-python3 ?

Reply:
nope, it's on out aptly server under govuk-python, tested it out on Integration and it works.

Reply:
ok cool